eth2util/keystore/keystore.go

```diff
@@ -108,8 +108,8 @@ func LoadKeys(dir string) ([]*bls_sig.SecretKey, error) {
 	return resp, nil
 }
 
-// keystore json file representation as a Go struct.
-type keystore struct {
+// Keystore json file representation as a Go struct.
+type Keystore struct {
```

Comment:
Best not refactor other packages just for testing.

Reply:
it is better code and it is already done and very short. In general I agree of not getting into other stuff outside the PR goal